watchtower/wtclient/client.go

```diff
@@ -317,7 +317,7 @@ func New(config *Config) (*TowerClient, error) {
 	c := &TowerClient{
 		cfg:               cfg,
 		log:               plog,
-		pipeline:          newTaskPipeline(),
+		pipeline:          newTaskPipeline(plog),
```

Comment:
There's a call to the package-level `log` variable on line 472 that should be re-written.

Reply:
good catch, fixed!